dashboard-frontend/src/pages/StudiesPage/useStudies.tsx

```diff
@@ -0,0 +1,95 @@
+import { useInfiniteQuery } from 'react-query';
+import { StudiesApiResponse } from '../../types/study';
+
+const studiesPerPage = 10;
+const defaultErrorMessage = 'Something went wrong while fetching the Studies.';
+
+type Cursor = string | null;
+
+const useStudies = () => {
+  const queryKey = 'studies';
+
+  const query = useInfiniteQuery<StudiesApiResponse, string, Cursor>(
+    queryKey,
+    fetchStudies,
+    {
+      getFetchMore: lastPage => lastPage.pagination.nextCursor,
+    }
+  );
+
+  return {
+    query,
+    queryKey,
+    studiesPerPage,
+    studies: (query.data || [])
+      .flatMap(page => page)
+      .map(studyResponse => studyResponse.data)
+      .flatMap(studyData => studyData),
+    errorMessage: query.error?.message || defaultErrorMessage,
+  };
+};
+
+/**
+ * TODO:
```

Comment:
Great job implementing exactly what's needed for now with reasonable TODO comments for how it should be generalized/abstracted when the time comes.